mod/content.php

```diff
@@ -197,7 +197,7 @@ function content_content(&$a, $update = 0) {
 	}
 
 	if($conv) {
-		$myurl = $a->get_baseurl() . '/profile/'. $a->user['nickname'];
+		$myurl = App::get_baseurl() . '/profile/'. $a->user['nickname'];
```

Comment:
In my develop branch, there are 372 occurances of `$a->get_baseurl()` left ... Let's bring them all down (I don't prefer it, because the method uses `$this` a lot) and change method signature to "public static".